### DISCUSSION TITLE
[A-L] (2023/24) Foglio 12 - Esercizio 1

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#336] [Esercizi - Viaggi/Piazza] [answered]
![Screenshot 2023-11-28 131648](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/d913727e-0386-4b01-b2f9-f817c73dfcff)

> Esercizio a cui fa riferimento: #97

**Comment by Elia-Belli** [comment]
Determinare le coordinate di $\underline e_2$ vuol dire trovare quei coefficienti $\alpha_1,\alpha_2,\alpha_3$ che utilizzati nella combinazione lineare dei tre vettori della base restituiscono $\underline e_2=\alpha_1v_1+\alpha_2v_2+\alpha_3v_3$; per farlo basta risolvere il seguente sistema:

$$\begin{cases}\alpha_1+3\alpha_2-\alpha_3=0\\
2\alpha_2+\alpha_2-\alpha_3=1\\
2\alpha_1-\alpha_2=0
\end{cases}\longrightarrow
\begin{pmatrix} 1 & 3 & -1\\
0 & -5 & 1\\
0 & 0 & -3/5
\end{pmatrix}
\begin{pmatrix}0\\
1\\
-7/5
\end{pmatrix}\longrightarrow\begin{cases} \alpha_1= -1/3\\
\alpha_2 = -2/3\\
\alpha_3= -7/3
\end{cases}$$

Quindi le coordinate di $\underline e_2$ nella base $(v_1,v_2,v_3)$ sono $(-1/3,-2/3,-7/3)$

**Comment by IonutCicio** [comment]
Analogo a @Elia-Belli

**Comment by CiottoloMaggico** [comment]
Analogo @Elia-Belli 
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/07449959-a367-4a61-82e8-e0d114189fd4)

**Comment by Elia-Belli** [comment, marked as answer]
# Soluzione Ufficiale
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/b5df733a-5bfa-4001-ad80-3ea0c8cde70f)
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/af528d4e-e4d2-47f6-bbdd-1debc4f75879)
